README.md

```diff
@@ -473,6 +473,14 @@ const container = awilix.createContainer()
 // Load our modules!
 container.loadModules([
   // Globs!
+  [
+    // to have different resolverOptions against single module.
```

Comment:
Change to `// To have different resolverOptions for specific modules` and I'll merge. 😄

Reply:
@jeffijoe done.